WordPress/WordPressTest/PostSyncStateViewModelTests.swift

```diff
@@ -1,66 +0,0 @@
-import Nimble
-import XCTest
-@testable import WordPress
-
-final class PostSyncStateViewModelTests: CoreDataTestCase {
```

Comment:
I think we can keep this test file, wdyt?

Reply:
It's testing the deprecated remote status values. It should ideally be rewritten, but I probably wouldn't invest time into it. It's simple enough and the test code complexity would be similar to the complexity of the code under test, which is not a good indicator.